projects/github.com/so-fancy/diff-so-fancy/package.yml

```diff
@@ -0,0 +1,19 @@
+distributable:
+  url: https://github.com/so-fancy/diff-so-fancy/archive/refs/tags/v{{version}}.tar.gz
+  strip-components: 1
+
+versions:
+  github: so-fancy/diff-so-fancy
+  strip: /^v/
+
+build:
+  script: |
+    chmod +x diff-so-fancy
+    mkdir -p "{{prefix}}/bin"
+    ln -sf "$PWD/diff-so-fancy" "{{prefix}}/bin"
```

Comment:
Something's definitely wrong here but I'm not sure what it is.

Reply:
That's hardcoding the build $PWD in the symlink. You want `mv diff-so-fancy "{{prefix}}/bin/"`.